### PR TITLE
feat(dlp): added a sample for dlp inspect bigquery send to scc

### PR DESCRIPTION
**Pull request by AarshD-crest**
## Description

- Added a sample for [dlp_inspect_bigquery_send_to_scc](https://cloud.google.com/dlp/docs/sending-results-to-scc.md#bigquery-data) with a unit test case.

Note: Before submitting a pull request, please open an issue for discussion if you are not associated with Google.

## Checklist
- [X] I have followed [Contributing Guidelines from CONTRIBUTING.MD](https://github.com/GoogleCloudPlatform/golang-samples/blob/main/CONTRIBUTING.md)
- [X] **Tests** pass:   `go test -v ./..` (see [Testing](https://github.com/GoogleCloudPlatform/golang-samples/blob/main/CONTRIBUTING.md#testing))
- [X] **Code formatted**:   `gofmt` (see [Formatting](https://github.com/GoogleCloudPlatform/golang-samples/blob/main/CONTRIBUTING.md#formatting))
- [X] **Vetting** pass:   `go vet` (see [Formatting](https://github.com/GoogleCloudPlatform/golang-samples/blob/main/CONTRIBUTING.md#formatting))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/golang-samples/blob/main/.github/CODEOWNERS) with the codeowners for this sample
- [ ] This sample adds a new **Product API**, and I updated the [Blunderbuss issue/PR auto-assigner](https://github.com/GoogleCloudPlatform/golang-samples/blob/main/.github/blunderbuss.yml) with the codeowners for this sample
- [X] Please **merge** this PR for me once it is approved

### CODE REVIEW

File: dlp/snippets/inspect/inspect_test.go [hunk 1]
```diff
@@ -681,3 +684,207 @@ func TestInspectTableWithCustomHotword(t *testing.T) {
 		t.Errorf("TestInspectTableWithCustomHotword got %q, want %q", got, want)
 	}
 }
+
+func TestInspectDataStoreSendToScc(t *testing.T) {
```
Comment (annadata) [inline dlp/snippets/inspect/inspect_test.go:688]: Is this for a different sample?

**Review by annadata — APPROVED** [review]

File: dlp/snippets/inspect/inspect_test.go (hunk 1 above)
Comment (AarshD-crest) [reply dlp/snippets/inspect/inspect_test.go:688]: Yes, it's for a different sample but I think due to some unwanted commits or branch conflicts I have to add this one to this PR, Sorry for the confusion.

File: dlp/snippets/inspect/inspect_test.go [hunk 2]
```diff
@@ -681,3 +684,206 @@ func TestInspectTableWithCustomHotword(t *testing.T) {
...
+		log.Printf("[INFO] deleteJob:: error %v", err)
+		return err
+	}
+	log.Printf("[INFO] deleteJob:: error %v", err)
+
+	req := &dlppb.DeleteDlpJobRequest{
+		Name: jobName,
+	}
+	for {
+		abc, err := client.GetDlpJob(ctx, &dlppb.GetDlpJobRequest{
```
Comment (telpirion) [inline dlp/snippets/inspect/inspect_test.go:863]: question: should we put a timeout on this `Context` object? Or do we know that the `GetDlpJob()` method has a _very_ high reliability?

Comment (AarshD-crest) [reply dlp/snippets/inspect/inspect_test.go]: Added a new context with timeout.

File: dlp/snippets/inspect/inspect_test.go [hunk 3]
```diff
@@ -681,3 +684,208 @@ func TestInspectTableWithCustomHotword(t *testing.T) {
...
+		t.Errorf("InspectBigQuerySendToScc got %q, want %q", got, want)
+	}
+}
+
+const (
+	dataSetID = "dlp_test_dataset"
+	tableID   = "dlp_inspect_test_table_table_id"
+)
+
+func TestInspectBigQuerySendToScc(t *testing.T) {
```
Comment (telpirion) [inline dlp/snippets/inspect/inspect_test.go:710]: issue: one test per file, please.

See https://googlecloudplatform.github.io/samples-style-guide/#dedicated-testing-per-sample

**Review by telpirion — CHANGES_REQUESTED** [review]

File: dlp/snippets/inspect/inspect_test.go (hunk 3 above)
Comment (AarshD-crest) [reply dlp/snippets/inspect/inspect_test.go:710]: Waiting for your reply on https://github.com/GoogleCloudPlatform/golang-samples/pull/3242#discussion_r1313028235

Comment (AarshD-crest) [reply dlp/snippets/inspect/inspect_test.go]: Hey @telpirion Hi, I have split the test for this particular sample into a separate file.

**Review by telpirion — APPROVED** [review]

**Pull request merged**